lib/shared/messages.js

```diff
@@ -34,7 +34,7 @@ module.exports = {
 
     flashComplete: _.template([
       '<%= imageBasename %> was successfully written to',
-      '<%= drive.description %> (<%= drive.name %>)'
+      '<%= drive.description %> (<%= drive.device %>)'
```

Comment:
Won't this change also affect the GUI too? I assume `drive.name` was previously used here for good reasons?
(I can never remember what the `.name` and `.device` fields map to for different OSes)